onnxruntime/core/optimizer/qdq_transformer/selectors_actions/shared/utils.cc

```diff
@@ -50,8 +50,9 @@ static const OpVersionsAndSelector::OpVersionsMap GetUnaryOpVersionsMap() {
           {"Slice", {}},
           {"Softmax", {}},
           {"Sqrt", {}},
-          {"Tanh", {}}};
-}
+          {"Tanh", {}},
+          {"InstanceNormalization", {}}};
```

Comment:
This is necessary so that the nodes DQ -> InstanceNormalization -> Q are treated as a single node unit.